app/views/shared/_pii_review.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="pl5 pb3">
+  <div class="h6">
+    <%= t('idv.review.full_name') %>
+  </div>
+
+  <div class="h4 bold ico-absolute ico-absolute-success">
+    <%= "#{pii[:first_name]} #{pii[:last_name]}" %>
```

Comment:
the string interpolation is kind of a slim thing because slim is so weird about whitespace. We could do this like?


```suggestion
    <%= pii[:first_name] %> <%= pii[:last_name] %>
```